chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -133,6 +133,7 @@
 <!ENTITY zotero.preferences.keys.newItem					"Create a New Item">
 <!ENTITY zotero.preferences.keys.newNote					"Create a New Note">
 <!ENTITY zotero.preferences.keys.toggleAllRead				"Mark All Feed Items As Read/Unread">
+<!ENTITY zotero.preferences.keys.toggleTabsMenu				"Toggle Tabs Menu">
```

Comment:
No more .dtd or .properties strings!

If we want to add this to the prefs, we'll need to move these strings to preferences.ftl (or special-case this and the new zotero.properties ones for now).

Also, while this does hide the menu too, "Toggle" doesn't seem right for a menu. I think we can just say "Show Tabs Menu".